src/lib/core/datetime/index.ts

```diff
@@ -20,7 +20,10 @@ export * from './native-date-formats';
 
 
 @NgModule({
-  providers: [{provide: DateAdapter, useClass: NativeDateAdapter}],
+  providers: [
+    {provide: DateAdapter, useClass: NativeDateAdapter},
+    {provide: MAT_DATE_LOCALE, useExisting: LOCALE_ID}
```

Comment:
I think this should go in the `MdDatepickerModule`

Reply:
@jelbourn can you think of a good way to avoid putting this in `NativeDateModule`, `MomentDateModule`, etc? I was thinking maybe we could put it in MdCommonModule and have this module import that?

Reply:
I think it doesn't really belong in `MdCommonModule` since it is only used in a few components. You could create a separate `MdCommonDateModule`, but that also seems like overkill. Easier thing would probably be just to make a common provider, e.g.
```ts
export const MAT_DATE_LOCALE_PROVIDER = {provide: MAT_DATE_LOCALE, useExisting: LOCALE_ID};
```
Adding that is about the same as importing another module.